package.json

```diff
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-tokenization",
   "dependencies": {
-    "@digitalbazaar/bitstring": "^1.0.0",
+    "@digitalbazaar/bitstring": "^1.2.0",
```

Comment:
I can't comment on the `bedrock-mongodb` peerDep, but I think it should be `^8.0.1`

Reply:
My code is already showing 1.2.0 ...  I'll see if this goes away when I push other changes.